js/apps/admin-ui/src/authentication/components/modals/AddStepModal.tsx

```diff
@@ -95,7 +95,8 @@ export const AddStepModal = ({ name, type, onSelect }: AddStepModalProps) => {
       localeSort(providers ?? [], mapByKey("displayName"))
         .filter(
           (p) =>
-            p.displayName?.includes(search) || p.description?.includes(search),
+            p.displayName?.toLowerCase().includes(search.toLowerCase()) ||
+            p.description?.toLowerCase().includes(search.toLowerCase()),
```

Comment:
Just realized, for performance reasons we might want to normalize the `search` outside of this `filter()` so it doesn't run twice  on every operation. Also we can use destructuring to reduce the amount of repetition.

```ts
const page = useMemo(
  () => {
    const normalizedSearch = search.trim().toLowerCase();

    return localeSort(providers ?? [], mapByKey("displayName"))
      .filter(({ displayName, description }) => (
        displayName?.toLowerCase().includes(normalizedSearch) ||
        description?.toLowerCase().includes(normalizedSearch)
      ))
      .slice(first, first + max + 1)
  },
  [providers, search, first, max],
);
```

Reply:
done!